src/base/bittorrent/session.h

```diff
@@ -51,14 +51,6 @@
 #include "sessionstatus.h"
 #include "torrentinfo.h"
 
-#if !defined(Q_OS_WIN) || (LIBTORRENT_VERSION_NUM >= 10212)
-#define HAS_HTTPS_TRACKER_VALIDATION
-#endif
-
-#if ((LIBTORRENT_VERSION_NUM >= 10212) && (LIBTORRENT_VERSION_NUM < 20000)) || (LIBTORRENT_VERSION_NUM >= 20002)
```

Comment:
FYI: compilation will fail if using libtorrent v2.0.0 or v2.0.1

Reply:
>compilation will fail if using libtorrent v2.0.0 or v2.0.1

Yes, since minimal supported version of libtorrent2 is raised too, as indicated in the PR title:
>Raise minimum libtorrent version to 1.2.12 (2.0.2)